### PR TITLE
Fix/menu bug fix/christina

### PR DESCRIPTION
**Pull request by c-melchor**
App bar and side drawer were not correctly rendering on smaller screen.
This branch fixes that bug and allows the menu icon to render and be clicked.
Breakpoint based on iPad px.

### CODE REVIEW

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 1]
```diff
@@ -104,11 +105,25 @@ const useStyles = makeStyles(theme => ({
...
+      () => {
+        const mobile = window.innerWidth < 769;
+        if (mobile !== mobile) {
+          setMobile(mobile);
+        }
+      },
+      false
+    );
+  }, [mobile]);
+
```
Comment (JusticeMatthew) [inline src/components/pages/SideDrawer/SideDrawer.js:126]: I would definitely recommend adding a cleanup function to this useEffect so we don't just leave that event listener on the window :)

**Review by JusticeMatthew — APPROVED** [review]
Great work!

**Comment by c-melchor** [comment]
Fixed a bug I had with the ability to click on items when side drawer was closed. Ready for review.

File: src/components/pages/SideDrawer/SideDrawer.js (hunk 1 above)
Comment (c-melchor) [reply src/components/pages/SideDrawer/SideDrawer.js:125]: Ok, I can add this tomorrow. Thanks.

**Review by ndossett — APPROVED** [review]
Nice job on the fix. Even though we expect this to be desktop first, it's important to have responsiveness just in case!

**Pull request merged**